exp/templates/exp/_navigation.html

```diff
@@ -7,48 +9,48 @@
-            <li><a href="{% url 'exp:lab-list' %}">Manage Labs</a></li>
-            <li><a href="{% url 'exp:participant-list' %}">View Participants</a></li>
-            {% if perms.accounts.can_view_analytics %}
-                <li><a href="{% url 'exp:study-participant-analytics' %}">Recruitment Stats</a></li>
-            {% endif %}
-            <li class="dropdown">
-                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
-                    Help <span class="caret"></span>
-                </a>
-                <ul class="dropdown-menu">
-                  <li><a href="https://lookit.readthedocs.io/" target="_blank">Docs</a></li>
-                </ul>
-            </li>
-            
-            <li role="separator" class="divider-vertical"></li>
-            <li class="dropdown">
-                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
-                    {{ request.user.identicon_small_html }} {{ request.user.get_short_name }} <span class="caret"></span>
-                </a>
-                <ul class="dropdown-menu">
-                    <li><a href="{% url 'accounts:manage-account' %}">Manage Account</a></li>
-                    <li><a href="{% url 'logout' %}">Logout</a></li>
-                </ul>
-            </li>
+        {% if perms.accounts.can_view_analytics %}
+        {% nav_item request 'exp:study-participant-analytics' 'Recruitment Stats' %}
+        {% endif %}
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+            Help <span class="caret"></span>
+          </a>
+          <ul class="dropdown-menu">
+            <li><a href="https://lookit.readthedocs.io/" target="_blank">Docs</a></li>
+          </ul>
+        </li>
+
+        <li role="separator" class="divider-vertical"></li>
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+            {{ request.user.identicon_small_html }} {{ request.user.get_short_name }} <span class="caret"></span>
+          </a>
+          <ul class="dropdown-menu">
+            <li><a href="{% url 'accounts:manage-account' %}">Manage Account</a></li>
+            <li><a href="{% url 'logout' %}">Logout</a></li>
+          </ul>
+        </li>
         {% else %} {# /if request.user.is_authenticated #}
-            {% if 'login' not in request.path %}
-                <li><a href="{% url 'login' %}?next={{ request.get_full_path.urlencode }}">Login</a></li>
-            {% endif %}
+        {% if 'login' not in request.path %}
+        <li><a href="{% url 'login' %}?next={{ request.get_full_path.urlencode }}">Login</a></li>
+        {% endif %}
```

Comment:
why is the formatting now doing this? (don't go too deep down this rabbit hole, just curious if there's an obvious answer)

Reply:
I don't know why the default formatter for vscode does anything.  But to me, it looks like it just removed the whitespace in front of those lines.  I can put the whitespace back if prefer.  

As a side note, I wish I could find a reasonable universal Django template format software.

Reply:
no worries, just curious!